### PR TITLE
[v0.9.1] Remove special versioning suffix rules for CUDA 10.2

### PR DESCRIPTION
**Pull request by seemethere**
Coincides with https://github.com/pytorch/pytorch/pull/53133

Signed-off-by: Eli Uriegas <eliuriegas@fb.com>

### CODE REVIEW

**Review by walterddr — APPROVED** [review]

File: packaging/pkg_helpers.bash [hunk 1]
```diff
@@ -36,9 +36,7 @@ setup_cuda() {
   # Wheel builds need suffixes (but not if they're on OS X, which never has suffix)
   if [[ "$BUILD_TYPE" == "wheel" ]] && [[ "$(uname)" != Darwin ]]; then
     # The default CUDA has no suffix
```
Comment (xuzhao9) [inline packaging/pkg_helpers.bash:38]: Should we remove this comment as well since all CUDA versions have a suffix now?

**Review by xuzhao9 — APPROVED** [review]

**Review by malfet — APPROVED** [review]

File: packaging/pkg_helpers.bash (hunk 1 above)
Comment (malfet) [inline packaging/pkg_helpers.bash:38]: Agree with @xuzhao9, we should remove the suffix
```suggestion
```

File: packaging/pkg_helpers.bash [hunk 2]
```diff
@@ -35,10 +35,7 @@ setup_cuda() {
   export WHEEL_DIR=""
   # Wheel builds need suffixes (but not if they're on OS X, which never has suffix)
   if [[ "$BUILD_TYPE" == "wheel" ]] && [[ "$(uname)" != Darwin ]]; then
-    # The default CUDA has no suffix
-    if [[ "$CU_VERSION" != "cu102" ]]; then
-      export PYTORCH_VERSION_SUFFIX="+$CU_VERSION"
-    fi
+    export PYTORCH_VERSION_SUFFIX="+$CU_VERSION"
```
Comment (fmassa) [inline packaging/pkg_helpers.bash:38]: Should we cherry-pick this change to the master branch?

Comment (seemethere) [reply packaging/pkg_helpers.bash:38]: Yes, I'll make follow up PRs for both things here

**Pull request merged**